IntroToCSharp/Pages/Activities/Booleans.razor

```diff
@@ -0,0 +1,111 @@
+@inherits CaptainCoder.Page
+@page "/activities/booleans"
+
+
+<h1>Booleans</h1>
+<Section Title="What are booleans?">
+    <p>
+        In C# a boolean or a “bool”, is a value that attempts to answer one of the simplest questions in C#.
```

Comment:
C# only refers to them as bool never boolean

Reply:
You are correct. However, I think it is important to use the full word `boolean` when we talk about them outside of code.